<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>416</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>664</width>
    <height>416</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>664</width>
    <height>416</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>RFID cloner</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #57877E, stop:1 #1C5C53);
</string>
  </property>
  <property name="iconSize">
   <size>
    <width>664</width>
    <height>416</height>
   </size>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openPortButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(223, 167, 26, 250);
	border-radius: 7px;
}

QPushButton:pressed {
    background-color:  rgb(230, 190, 140, 240);
}</string>
    </property>
    <property name="text">
     <string>Открыть порт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cloneButton">
    <property name="geometry">
     <rect>
      <x>351</x>
      <y>71</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>San Francisco</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(223, 167, 26, 250);
	border-radius: 7px;
}

QPushButton:pressed {
    background-color:  rgb(230, 190, 140, 240);
}</string>
    </property>
    <property name="text">
     <string>Клонировать карту</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closePortButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(223, 167, 26, 250);
	border-radius: 7px;
}

QPushButton:pressed {
    background-color:  rgb(230, 190, 140, 240);
}</string>
    </property>
    <property name="text">
     <string>Закрыть порт</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 150, 255, 128)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="writeCard">
    <property name="geometry">
     <rect>
      <x>351</x>
      <y>143</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(223, 167, 26, 250);
	border-radius: 7px;
}

QPushButton:pressed {
    background-color:  rgb(230, 190, 140, 240);
}</string>
    </property>
    <property name="text">
     <string>Записать UID на карту</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testButton">
    <property name="geometry">
     <rect>
      <x>351</x>
      <y>286</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(223, 167, 26, 250);
	border-radius: 7px;
}

QPushButton:pressed {
    background-color:  rgb(230, 190, 140, 240);
}</string>
    </property>
    <property name="text">
     <string>Тестировать</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>281</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255, 200);
border-radius: 7px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="writeToNewCardButton">
    <property name="geometry">
     <rect>
      <x>351</x>
      <y>214</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(223, 167, 26, 250);
	border-radius: 7px;
}

QPushButton:pressed {
    background-color:  rgb(230, 190, 140, 240);
}</string>
    </property>
    <property name="text">
     <string>Записать дамп на новую карту</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
